build.json

```diff
@@ -42,15 +42,6 @@
         "net472",
         "net8.0"
       ]
-    },
-    {
-      "Name": "Microsoft.SqlTools.SqlCore",
-      "Frameworks": [
-        "net6.0",
```

Comment:
We still need the net6 one for now - will it do that automatically since it's the only target framework?

If you have a test build you did for comparison that'd be great.

Reply:
Yup, it still builds .NET 6; I only removed it from the `FxBuildProjects` section.

The nuget produced via the [PR build](https://msdata.visualstudio.com/SQLToolsAndLibraries/_build/results?buildId=135257542&view=artifacts&pathAsName=false&type=publishedArtifacts) still has .NET 6 binaries in it, but I've kicked off a [Buddy build](https://msdata.visualstudio.com/SQLToolsAndLibraries/_build/results?buildId=135351899&view=results) for even more confirmation.

Reply:
Ah, thanks! Looks good!

Reply:
Buddy build also looks good.